test/data/base/03-include-in-define.yaml

```diff
@@ -0,0 +1,9 @@
+otk.version: "1"
+
+otk.define.name:
+  otk.include: "03-include-in-define/example.yaml"
```

Comment:
I think it's probably a good idea to put this under a subtree to test that this also works, something like:
```yaml
diff --git a/test/data/base/03-include-in-define.yaml b/test/data/base/03-include-in-define.yaml
index f750223..1f6dece 100644
--- a/test/data/base/03-include-in-define.yaml
+++ b/test/data/base/03-include-in-define.yaml
@@ -1,8 +1,9 @@
 otk.version: "1"
 
-otk.define.name:
-  otk.include: "03-include-in-define/example.yaml"
+otk.define:
+  subkey:
+    otk.include: "03-include-in-define/example.yaml"
 
 pipelines:
-  - foo: ${foo}
-    bar: ${bar}
+  - foo: ${subkey.foo}
+    bar: ${subkey.bar}
```

Reply:
I've added a subkey in a separate commit, both an include at the top-level under `otk.define` and the subkey.